src/test/java/task40/SeleniumEasyTests.java

```diff
@@ -0,0 +1,204 @@
+package task40;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.openqa.selenium.Alert;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.firefox.FirefoxDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.Select;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+
+import java.time.Duration;
+import java.util.*;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class SeleniumEasyTests {
+
+    private WebDriver driver;
+    private WebDriverWait wait;
+
+    @BeforeEach
+    public void setUp(){
+        WebDriverManager.firefoxdriver().setup();
+        driver = new FirefoxDriver();
+        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(5));
+        wait = new WebDriverWait(driver, Duration.ofSeconds(5));
+    }
+
+    @AfterEach
+    public void cleanUp(){
+        driver.quit();
+    }
+
+    @Test
+    public void Point5Test() {
+        By multiselectValuesLocator = By.cssSelector("#multi-select option");
+
+        driver.get("https://demo.seleniumeasy.com/basic-select-dropdown-demo.html");
+
+        List<String> countries = this.getRandomCountries(3);
+
+        for (String country: countries) {
+            WebElement element = driver.findElements(multiselectValuesLocator)
+                    .stream().filter(x -> x.getText().equals(country)).findFirst().orElse(null);
```

Comment:
better to throw  some exception in orElse block in order to avoid NullPointer exception